resources/lib/builds.py

```diff
@@ -228,7 +228,7 @@ def __iter__(self):
         if self.CSS_CLASS is not None:
             args.append(self.CSS_CLASS)
 
-        self.build_re = re.compile(self.BUILD_RE.format(arch=arch))
+        self.build_re = re.compile(self.BUILD_RE.format(release=openelec.OS_RELEASE['NAME'],arch=arch))
```

Comment:
How about a `dist` function in the `openelec` module which just returns `OS_RELEASE['NAME']`. That would read better I think.